### PR TITLE
HLE/Applets: Stub Mint (eShop) Applet

### PR DESCRIPTION
**Pull request by mailwl**
this PR let start Phoenix Wright - Dual Destinies

### CODE REVIEW

File: src/core/hle/applets/mint.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+class Mint final : public Applet {
+public:
+    explicit Mint(Service::APT::AppletId id) : Applet(id) {}
+
+    ResultCode ReceiveParameter(const Service::APT::MessageParameter& parameter) override;
+    ResultCode StartImpl(const Service::APT::AppletStartupParameter& parameter) override;
+    void Update() override;
+
+private:
+    /// This SharedMemory will be created when we receive the LibAppJustStarted message.
```
Comment (Subv) [inline src/core/hle/applets/mint.h]: LibAppJustStarted doesn't exist anymore.

File: src/core/hle/applets/mint.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+ResultCode Mint::ReceiveParameter(const Service::APT::MessageParameter& parameter) {
+    if (parameter.signal != static_cast<u32>(Service::APT::SignalType::Request)) {
+        LOG_ERROR(Service_APT, "unsupported signal %u", parameter.signal);
+        UNIMPLEMENTED();
+        // TODO(Subv): Find the right error code
+        return ResultCode(-1);
+    }
+
+    // The LibAppJustStarted message contains a buffer with the size of the framebuffer shared
```
Comment (Subv) [inline src/core/hle/applets/mint.cpp]: LibAppJustStarted doesn't exist anymore

File: src/core/hle/applets/mint.h (hunk 1 above)
Comment (mailwl) [inline src/core/hle/applets/mint.h]: I don't understand, what to fix? It's just copy/paste from ErrEula (most generic applet)

Comment (Subv) [inline src/core/hle/applets/mint.h]: Update the comment, don't just copypaste stale information.

**Comment by mailwl** [comment]
done. i also can change outdated comments in other applets as well

**Pull request merged**